server/models/current-year.js

```diff
@@ -0,0 +1,4 @@
+module.exports = (req, res, next) => {
+	res.locals.currentYear = new Date().getFullYear();
```

Comment:
Let's hope the servers have the right time set up 🤞

Reply:
Heh :) the email services (and apparently myFT) have been using this for a while and so there's prior proven use ☺️